src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpoint.java

```diff
@@ -160,6 +164,7 @@ public CaseDetails findCaseDetailsForCitizen(
         @ApiResponse(code = 404, message = "No case found for the given ID"),
         @ApiResponse(code = 422, message = "Process could not be started")
     })
+    @LogAudit(operationType = OperationType.CREATE_CASE)
```

Comment:
I think we agreed we wouldn't log these start event operations:
```Hi @Mario We have few questions for Log and Audit. https://tools.hmcts.net/confluence/display/RCCD/CCD+Case+Log+And+Audit+Endpoints+Data+Availability+-+V1
1.) Do we need to log this one: Case access API - GET /caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/ids
2.) Do we need to log middle case creation steps, like Standard case API - GET /caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/event-triggers/{etid}/token  Those are trigerred on starting a new event (for Update or Create case), but not when we persist the case on the Submit.






Mario:house_with_garden:  12:32 PM
Hi @Rafal Kalita 1) no 2) no ```

Reply:
Same as other comments. Will clean up and will be in v1 PR.

Reply:
the reason I think it's best not to log the start of a case update it that it's better to have 1 log audit for each user operation. Because to update a case in CCD you need to call 2 endpoint, each user update would be result with 2 audit logs. Which is not nice